tests/gcp/hooks/test_kms.py

```diff
@@ -18,21 +18,32 @@
 # under the License.
 
 import unittest
-from base64 import b64encode
+from base64 import b64encode, b64decode
+from collections import namedtuple
 
 from airflow.gcp.hooks.kms import GoogleCloudKMSHook
 from tests.compat import mock
 
-BASE_STRING = 'airflow.contrib.hooks.gcp_api_base_hook.{}'
-KMS_STRING = 'airflow.gcp.hooks.kms.{}'
 
+Response = namedtuple("Response", ["plaintext", "ciphertext"])
 
-TEST_PROJECT = 'test-project'
-TEST_LOCATION = 'global'
-TEST_KEY_RING = 'test-key-ring'
-TEST_KEY = 'test-key'
-TEST_KEY_ID = 'projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}'.format(
-    TEST_PROJECT, TEST_LOCATION, TEST_KEY_RING, TEST_KEY)
+PLAINTEXT = b"Test plaintext"
+PLAINTEXT_b64 = b64encode(PLAINTEXT).decode("ascii")
+
+CIPHERTEXT_b64 = b64encode(b"Test ciphertext").decode("ascii")
+CIPHERTEXT = b64decode(CIPHERTEXT_b64.encode("utf-8"))
+
+AUTH_DATA = b"Test authdata"
+
+TEST_PROJECT = "test-project"
+TEST_LOCATION = "global"
+TEST_KEY_RING = "test-key-ring"
+TEST_KEY = "test-key"
+TEST_KEY_ID = "projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}".format(
+    TEST_PROJECT, TEST_LOCATION, TEST_KEY_RING, TEST_KEY
+)
```

Comment:
Google python clients often provide `path_for_something` methods that construct such uris. Usually it's good idea to use them because they are up to date (well, at least they try...). I am not sure if this could by used here:
https://googleapis.dev/python/cloudkms/latest/_modules/google/cloud/kms_v1/gapic/key_management_service_client.html#KeyManagementServiceClient.crypto_key_version_path

Reply:
In tests it is fine not to use it. This way we don't have to mock it, but when the library changes its behavior we can detect it.

Reply:
Those method's are usually static so no need for mocking.

Reply:
Due to the fact this string is used only in assertion it doesn't seem necessary to use mentioned method. I wanted to keep backward compatibility so the input to the functions (`key_name` parameter) should be string formatted by the user (so `crypto_key_path` and `crypto_key_version_path` methods from `KeyManagementServiceClient ` are not applicable here)